src/controllers/AdminUserManagementController.java

```diff
@@ -91,6 +91,14 @@ private void DisplaySuccessOnCreateAccount(UserData<?> user) {
             adminScreenView.showRegisterUserSuccess();
         }
     }
-
+    private boolean DeleteUserHelper(String username) {
+        if (patientManager.deleteUser(username)) {
+            return true;
+        } else if (doctorManager.deleteUser(username)) {
+            return true;
+        } else if (secretaryManager.deleteUser(username)) {
+            return true;
+        } else return adminManager.deleteUser(username);
+    }
```

Comment:
This method can be reduced using boolean logic.

Reply:
```
return blabla |
        bloblo |
        bleble;
```